app/Http/Controllers/ScheduledMessageController.php

```diff
@@ -92,13 +99,14 @@ public function update(ScheduledMessage $scheduled_message, Request $request)
         $scheduled_message->target_sms = $request->input('target_sms');
         $scheduled_message->target_twitter = $request->input('target_twitter');
         $scheduled_message->send_at = new Carbon($request->input('send_at'));
+        $scheduled_message->tags()->sync(array_keys($request->input('tags')));
```

Comment:
Same situation as above.

```suggestion
        $scheduled_message->tags()->sync($request->input('tags'));
```